modules/virt-configuring-vm-use-usb-device.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-VM-use-usb-device_{context}"]
+= Configuring a virtual machine connection to a USB device
+
+You can configure virtual machine (VM) access to a USB device. Doing so allows a guest to connect to actual USB hardware that is attached to an {product-title} node, as if the hardware and VM are physically connected.
+
+.Prerequisites
+
+* A cluster admin must enable the `hostDevices` feature gate on the HyperConverged (HCO) custom resource (CR).
+
+.Procedure
+
+. Locate the USB device by running the following command:
++
+[source,terminal]
+----
+$ oc /dev/serial/by-id/usb-VENDOR_device_name
+----
+
+. Open the HCO CR by running the following commmand:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
```

Comment:
The example on the next lines is then about a `VirtualMachineInstance` CR, not the HCO CR.

Reply:
@acardace, should I/how would I modify the YAML example to address Simone's comment?

Reply:
`$ oc edit vmi vmi-usb` because you're editing the VMI.

Reply:
TY!